plugins/auth/src/pam_password.cpp

```diff
@@ -271,11 +271,26 @@ namespace irods
 
                 log_auth::trace("redirecting call to CSP");
 
-                if (const auto ec = sslStart(host->conn); ec) {
-                    THROW(ec, "could not establish SSL connection");
-                }
+                // Need to enable SSL here if it is not already being used because the PAM password
+                // is forwarded to the provider in the clear.
+                // clang-format off
+                const bool using_ssl = 0 == std::strncmp(
+                        irods::CS_NEG_USE_SSL.c_str(),
+                        host->conn->negotiation_results, // NOLINT(cppcoreguidelines-pro-bounds-array-to-pointer-decay)
+                        MAX_NAME_LEN);
```

Comment:
Indentation?

Reply:
I was referring to the indentation appearing to use more than 4 spaces.
```cpp
const bool using_ssl = 0 == std::strncmp(
￼        irods::CS_NEG_USE_SSL.c_str(),
￼        host->conn->negotiation_results, // NOLINT(cppcoreguidelines-pro-bounds-array-to-pointer-decay)
￼        MAX_NAME_LEN);
```
VS
```cpp
const bool using_ssl = 0 == std::strncmp(
￼   irods::CS_NEG_USE_SSL.c_str(),
￼   host->conn->negotiation_results, // NOLINT(cppcoreguidelines-pro-bounds-array-to-pointer-decay)
￼   MAX_NAME_LEN);
```
The `NOLINTBEGIN`/`END` thing is not available in Clang 13's clang-tidy. I've been using those knowing that we'll be upgrading the compiler some time in the future.

Reply:
Ah, okay. Thanks. Will fix this up.